pkg/ruler/ruler.go

```diff
@@ -119,7 +119,7 @@ type Ruler struct {
 	notifierCfg *config.Config
 
 	scheduler *scheduler
-	workers   []worker
+	workers   *sync.WaitGroup
```

Comment:
Hrm, the naming is a little off here. I would expect this would have the workers. `workersWG` is better?